tomcat9x/src/integTest/groovy/com/bmuschko/gradle/tomcat/embedded/Tomcat9xServerIntegrationTest.groovy

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.bmuschko.gradle.tomcat.embedded
+
+import com.bmuschko.gradle.tomcat.embedded.fixture.EmbeddedTomcatIntegrationTest
+
+/**
+ * Tomcat 9x server test.
+ *
+ * @author Benjamin Muschko
+ * @author Keisuke Kato
+ */
+class Tomcat9xServerIntegrationTest extends EmbeddedTomcatIntegrationTest {
```

Comment:
All `EmbeddedTomcatIntegrationTest` long  very similar. We should try to unify the code here as well.

Reply:
@bmuschko I created `EmbeddedTomcat7xPlusIntegrationTest`.